src/Orchard.Web/Modules/Orchard.Users/Extensions/ModelStateDistionaryExtensions.cs

```diff
@@ -0,0 +1,15 @@
+using Orchard.Localization;
+using System.Collections.Generic;
+using Orchard.Mvc.Extensions;
+
+namespace System.Web.Mvc {
+    public static class ModelStateDictionaryExtensions {
+        public static void AddModelErrors(this ModelStateDictionary modelStateDictionary, IDictionary<string, LocalizedString> validationErrors) {
+            if (validationErrors.Count > 0) {
```

Comment:
And on a second look this won't even be called when there are no errors, so removed.